src/inspector_js_api.cc

```diff
@@ -32,6 +32,171 @@ std::unique_ptr<StringBuffer> ToProtocolString(Isolate* isolate,
   return StringBuffer::create(StringView(*buffer, buffer.length()));
 }
 
+class JSDispatcher;
+
+class JSDispatcherInterface : private AsyncWrap {
+ public:
+  JSDispatcherInterface(Environment* env, Local<Object> object)
+                      : AsyncWrap(env, object, PROVIDER_INSPECTORJSDISPATCHER),
+                        env_(env),
+                        dispatcher_(nullptr) {
+    Wrap(object, this);
+    env->SetMethod(object, "sendMessageToFrontend",
+                   JSDispatcherInterface::SendMessageToFrontend);
```

Comment:
This method should be set in the FunctionTemplate rather than on every invocation.